client/main.lua

```diff
@@ -869,7 +869,7 @@ CreateThread(function()
 								end
 							end
 
-							if displayText ~= "" then displayNUIText(displayText) end
+							if displayText ~= "" and closestDoor.data.hideLabel and closestDoor.data.hideLabel ~= true then displayNUIText(displayText) end --*
```

Comment:
You can remove the `~= true` part because the check before that already checks it